tools/subgraph-example/hardhat.config.ts

```diff
@@ -134,7 +226,7 @@ function updateStartBlock(dataSource: string, startBlock: number, hre: any) {
     },
   );
   const networks = JSON.parse(networksRaw);
-  networks[hre.network.name][dataSource].startBlock = startBlock;
+  networks[hre.network.name][dataSource].startBlock = startBlock || 1;
```

Comment:
With 0.40 this should now always be 0.
Unless you're testing against mainnet in the next 2 weeks this should be fine without the ` || 1`
Should leave in just want to make sure this is the case.

Reply:
You're right, updated.